### PR TITLE
Get library with BinDeps (Windows)

### PR DESCRIPTION
**Pull request by samuelpowell**
Implemented for 64-bit architectures only. Requires testing on

- [x] Windows

Other platforms to be added later

### CODE REVIEW

**Comment by samuelpowell** [comment]
@ReubenHill make sure this works for you? Will open issue for other platforms.

File: deps/build.jl [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# LibFTD2XX.jl
+
+using BinDeps
+
+Sys.WORD_SIZE == 64 || error("Build script configured only for x64")
+
+@BinDeps.setup
+
+function validate_libFTD2XX_version(name, handle)
```
Comment (ReubenHill) [inline deps/build.jl:9]: The `name` argument doesn't seem to be used in this function. What's it for?

Comment (ReubenHill) [reply deps/build.jl:9]: Nevermind, this is part of the BinDeps api

Comment (samuelpowell) [reply deps/build.jl:9]: Exactly

**Review by ReubenHill — APPROVED** [review]
Confirmed working on windows x64.

**Pull request merged**